### PR TITLE
Transactions: add response code and response message (GWY-4425)

### PR DESCRIPTION
**Pull request by dkotin**
Resolves: GWY-4425

### CODE REVIEW

File: src/Entities/AvsResponse.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Rebilly\Entities;
+
+use Rebilly\Rest\Entity;
+
+/**
+ * Class Gateway
+ */
+final class AvsResponse extends Entity
```
Comment (slavcodev) [inline src/Entities/AvsResponse.php]: It's not an entity, this object does not have `id`. You should use `Resource` in such classes.

File: src/Entities/AvsResponse.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rebilly\Entities;
+
+use Rebilly\Rest\Entity;
+
+/**
+ * Class AvsResponse.
+ */
+final class AvsResponse extends Entity
```
Comment (slavcodev) [inline src/Entities/AvsResponse.php]: It's not an entity, this object does not have id. You should use Resource in such classes.

File: src/Entities/Transaction.php [hunk 3]
```diff
@@ -155,7 +155,11 @@ public function getPaymentCard()
      */
     public function getGateway()
     {
-        return new Gateway($this->getAttribute('gateway'));
+        if ($this->getAttribute('gateway') === null) {
```
Comment (slavcodev) [inline src/Entities/Transaction.php]: Wouldn't better the ternary operator here?

**Comment by dkotin** [comment]
no more objections?

**Pull request merged**